Controllers/Backend/MollieOrders.php

```diff
@@ -65,16 +75,21 @@ public function shipAction()
             $mollieOrder = $this->apiClient->orders->get($mollieId);
             $errorMessage = '';
 
-            if (empty($mollieOrder))
+            if ($mollieOrder === null) {
```

Comment:
thx for touching that code. makes it better. you could indeed do an "instance of Order" check to even increase it a bit more if you want

Reply:
I'd prefer to leave this as a null check. instanceof makes the code harder to read and should always be returned by the `$this->apiClient->get()` method

Reply:
i generally prefere comparing to what we actually need and not that its not just null.
i cannot agree with harder to read, for me its a better stability in the end, but its ok you can leave it